test/python/test_envoy_config_template.yaml

```diff
@@ -0,0 +1,220 @@
+# this is just @envoy-mobile//library/common:config_template.c
+# except it doesn't use TLS
```

Comment:
Cleartext support is now present in the main config. Can we get rid of this and use that instead?